src/CoreServices/FSEvents.cs

```diff
@@ -360,6 +409,39 @@ public void Invalidate ()
 				return FSEventStreamGetLatestEventId (handle);
 			}
 		}
+
+		[DllImport (Constants.CoreServicesLibrary)]
+		static extern ulong FSEventStreamGetDeviceBeingWatched (IntPtr handle);
+
+		public ulong GetDevice ()
+		{
+			CheckDisposed ();
+			return FSEventStreamGetDeviceBeingWatched (handle);
+		}
+
+		[DllImport (Constants.CoreServicesLibrary)]
+		static extern bool FSEventStreamSetExclusionPaths (IntPtr handle, IntPtr pathsToExclude);
+
+		public bool SetExclusionPaths (NSArray pathsToExclude)
+		{
+			if (pathsToExclude == null) {
+				throw new ArgumentNullException ("pathsToExclude");
+			}
+			CheckDisposed ();
+			return FSEventStreamSetExclusionPaths (handle, pathsToExclude.Handle);
+		}
+
+		[DllImport (Constants.CoreServicesLibrary)]
+		static extern bool FSEventStreamSetDispatchQueue (IntPtr handle, DispatchQueue queue);
```

Comment:
that's documented as returning `void`
https://developer.apple.com/documentation/coreservices/1444164-fseventstreamsetdispatchqueue?language=objc